dashboard/urls.py

```diff
@@ -45,7 +47,9 @@
     url(r'^update_groups', cron.update_groups, name='update_groups'),
     url(r'^submission', cron.submission, name='submission'),
     url(r'^weight_consideration', cron.weight_consideration, name='weight_consideration'),
-
+    url(r'^testloader', TemplateView.as_view(
```

Comment:
This is good. I need to move other templates here following the TemplateView format. Nice work!

Reply:
Yeah this pattern seemed pretty easy to just display templates, we can re-organize the other templates on a future PR.

Reply:
I think there's some other refactoring that would be nice to do too, like breaking up views.py a bit before it gets too large.